pkgs/development/python-modules/helium/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  buildPythonPackage,
+  python3Packages,
+  fetchPypi,
+}:
+
+buildPythonPackage rec {
+  pname = "helium";
+  version = "5.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-cpnJJVRlPOpftoakyzc1ZsmCr2yqE/rv2qP8dc89ngY=";
+  };
+
+  patches = [
+    ./fix_helium_firefox_profile.patch # see https://github.com/mherrmann/helium/pull/132
+  ];
```

Comment:
you can actually fetch this directly from GitHub: [https://patch-diff.githubusercontent.com/raw/mherrmann/helium/pull/132.patch](https://patch-diff.githubusercontent.com/raw/mherrmann/helium/pull/132.patch) and apply that way, it is preferred to including the patch here.